submissions/trostinetska/hooli-style-popup/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en-US">
+<head>
+    <meta charset="utf-8">
+    <meta name="author" content="Larysa Trostinetska">
+    <meta name="description" content="popup">
+    <meta name="viewport" content="width=device-width, initial-scale=1"/>
+    <title>Popup</title>
+    <link rel="stylesheet" href="styles.css" type="text/css">
+</head>
+<body>
+    <header>
+        <nav class="navbar">
+            <ul class="crumbs">
+                <li class="crumb"><a href="#" tabindex="0">Mail</a></li>
+                <li class="crumb"><a href="#" tabindex="0">Images</a></li>
+                <li class="crumb">
+                <input type="checkbox" id="checkbox_toggle" name="checkbox_toggle" class="check_input" tabindex="0">
+                <label for="checkbox_toggle" class="check" >
+                    <img src="./icons/popup-button.png" alt="start-menu">
+                </label>
+                    <nav class="popup">
+                        <ul class="popup_list">
+                            <li class="popup_item"><a class="popup_link" href="#" tabindex="0"><img class="icon" src="./icons/calendar.png" alt="calendar"><p>Calendar</p></a></li>
```

Comment:
[Read this article](https://htmlacademy.ru/blog/articles/alt-text) to find out how and when populate `alt` attribute